app/services/DriveService.scala

```diff
@@ -26,30 +26,24 @@ object DriveService {
   lazy val driveClient: DriveClient = new DriveClient.Builder(httpTransport, jacksonFactory, scopedCredential)
     .build()
 
-  def getAllRecords: List[ResearchRecord] = {
+  def getAllApiFiles: Either[String, List[File]] = {
     val request = driveClient.files().list()
       .setCorpora("teamDrive")
       .setIncludeTeamDriveItems(true)
       .setSupportsTeamDrives(true)
       .setFields("*")
       .setTeamDriveId(Config.teamDriveId)
 
-    request.execute().getFiles.asScala.toList
-      .map(file =>
-        // transformToResearchRecord(extractRawFile(file)) // define functions with def
-        (extractRawFile andThen transformToResearchRecord)(file) // define functions with val
-    )
+    Try(request.execute().getFiles.asScala.toList).toEither.left.map { exception =>
+      exception.getMessage
```

Comment:
Be worth having a think about what you'd like to reveal to users here! Is there something more user friendly you could say? This is fine for now though because we haven't talked about logging yet.